java/compaction/compaction-core/src/main/java/sleeper/compaction/job/CompactionJob.java

```diff
@@ -22,8 +22,11 @@
 import java.util.stream.Collectors;
 
 /**
- * Contains the definition of a compaction job, including the id of the job,
- * a list of the input files, the output file or files, and the id of the partition
+ * Contains the definition of a compaction job. This includes the ID of the job,
+ * a list of the input files, the output file or files, and the ID of the partition.
```

Comment:
A compaction job can only have 1 output file now since we removed the concept of splitting compaction jobs.

Reply:
Updated, thanks